docs/cloud/index.md

```diff
@@ -4,50 +4,28 @@ title: Cloud Services
 
 # Overview
 
-CSC offers a variety of cloud computing services: Pouta services (cPouta and ePouta) and Rahti container cloud service.
-
-Pouta services are provided in a Infrastructure-as-a-Service model. Using Pouta services, you can can take control of your IT infrastructure, manage it in a flexible fashion, and automate it as you require. You can launch and manage virtual machines, deploy storage, and create networks for your needs. cPouta is targeted towards generic cloud needs, while ePouta is mainly used for handling sensitive data. Taking ePouta into use requires more set-up and always involves your organization's IT department.
-
-Rahti makes the deployment of applications a breeze. This is a higher level service than Pouta services, and Rahti automatically takes care of a lot of the lower level management, so that you don't have to. Rahti is based on Red Hat's OKD, which is a distribution of Kubernetes. It uses Docker containers to run applications.[^1]
+CSC Cloud Services consists of Pouta hardware virtualisation service, Rahti application containerisation service, Pukki cloud database service and CSC Notebooks web application course hosting service. CSC hosts all services itself, with data centres in Finland.
 
 ### Pouta
 
-With Pouta services, you can can take control of your IT infrastructure, manage it in a flexible fashion, and automate it as you require. You can launch and manage virtual machines, deploy storage, and create networks for your needs. This infrastructure can be used for anything you need or can imagine.[^2]
-
-<iframe
-    width="400"
-    height="300"
-    srcdoc="https://www.youtube.com/embed/2opuVIBSPgw"
-    title="CSC's Pouta Cloud Services"
-    frameborder="0"
-    allow="accelerometer; autoplay; clipboard-write; encrypted-media;gyroscope; picture-in-picture"
-    allowfullscreen>
-</iframe>
+The Pouta service is a cloud computing infrastructure based on the OpenStack Rocky release. It provides virtual computer hardware platforms, storage and computer network resources.
```

Comment:
(no strong opinion) I would not mention "Rocky release", I think it will confuse more than it will help.

Reply:
Removed.